src/main/java/net/yogstation/yogbot/listeners/channel/MemesChannel.kt

```diff
@@ -20,9 +20,9 @@ class MemesChannel(channelsConfig: DiscordChannelsConfig) : AbstractChannel(chan
 			message.embeds.size > 0 ||
 			memetypes.contains(message.content.split(".").last())
 			) {
-			// Java is strange with unicode in strings, this is thumbs up and down emoji
-			return message.addReaction(ReactionEmoji.unicode("\uD83D\uDC4D"))
-				.and(message.addReaction(ReactionEmoji.unicode("\uD83D\uDC4E")))
+			// Java is strange with unicode in strings, this is horse and fish emoji
```

Comment:
```suggestion
			// Java is strange with unicode in strings, this is 🐎 and 🐟emoji
```

Emojis are allowed in Java :)